specs/packagemanager/PackageManagement.md

```diff
@@ -145,7 +148,23 @@ not ready can include:
 Is\*Ready() methods are a quick test to determine if more (costly) work is needed before the target
 can be used.
 
-## 3.3. Ensure\*Ready()
+## 3.3. Is\*ReadyOrNewerAvailable()
+
+Is\*Ready() methods determine if the target is installed and ready for use or a newer version is
+locally available. Reasons why a package is not ready can include:
+
+* The package is not present on the machine
+* The package is present on the machine but not registered for the user
+* The package is registered for the user but is not in a healthy status e.g. its
+  Package.Status=Tampered
+* The package is registered for the user but a newer version is available locally on the machine
+  e.g. a user has package v1 registered and Windows Update downloaded and staged v2 in the
+  background while app(s) are running using v1.
+
```

Comment:
If the package is (say) not healthy, but a newer version is downloaded and staged, then IsReadyOrNewerAvailable returns true?

This section is confusing because the API under discussion is Is\*ReadyOrNewerAvailable, but all the text talks about is the Ready part and not the NewerAvailable part.

Reply:
If package not health but newer version staged then IsReadyOrNewerAvailable=true.

Per previous comment the phrasing is wrong and being updated.